.github/workflows/probot-check-group.yml

```diff
@@ -11,13 +11,16 @@ concurrency:
 jobs:
   required-jobs:
     runs-on: ubuntu-latest
-    if: github.event.pull_request.draft == false
-    # if this timeout triggers, then the job needs to be manually restarted through the GitHub interface
-    timeout-minutes: 60
+    # FIXME: debugging
+    #if: github.event.pull_request.draft == false
+    timeout-minutes: 3  # in case something is wrong with the internal timeout
     steps:
-      - uses: Lightning-AI/probot@v1
+      - uses: Lightning-AI/probot@v2
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           job: check-group
-          interval: 180  # seconds
+          interval: 30  # seconds
+          timeout: 2  # minutes
+          maintainers: '@Lightning-AI/lai-frameworks'
+          owner: '@carmocca'
```

Comment:
This is so that I can add `if owner == "@carmocca": just_pass()` :)

And also to show:

```
Error: The timeout of 2 minutes has triggered but not all required jobs were passing. This job will need to be re-run to merge your PR. If you do not have write access to the repository you can ask @Lightning-AI/lai-frameworks to re-run it for you. If you have any other questions, you can reach out to @carmocca for help.
```